index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main-container">
+      <div class="header">
+        <div class="logo">
+          <div class="logo-image">
+            <img
+              src="https://th.bing.com/th/id/OIP.jtdO5VDiwFucIyW4UzS9rAHaB-?w=350&h=93&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+              alt=""
+            />
+          </div>
+          <div class="logo-text">Original Trombones</div>
+        </div>
+        <div class="navigation-links">
+          <ul>
+            <li>
+              <a href="#">Features</a>
+            </li>
```

Comment:
I think it would be better to use **nav** tag for professionalism and bring semantic meaning to your code

Reply:
Hi, @Samuelwanza!! 👋
- Here you have made a good suggestion politely with reasoning. 👍 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.